app/services/queries/weekly_metrics.rb

```diff
@@ -0,0 +1,34 @@
+module Queries
+  class WeeklyMetrics < BaseService
```

Comment:
Could this and `DailyMetrics` inherit from another common superclass and put some shared code over there?

Reply:
i think both classes has some different important things like the constructor and the range of the time, but the way that they parse the data in fact was very similar for chartkick, so it was already moved to another class

Reply:
Yeah, I agree that it was a smart move. I just felt like many of these methods where exactly the same (`call`, `users_project`, `current_time`) so we could keep them in just one place.
But as it's not much, if you think readability is better this way, I'll leave it up to you whether create a superclass or not 👌